idioms-json-2.1-custom/incident-malware.json

```diff
@@ -3,29 +3,49 @@
     "objects": [
         {
             "created": "2014-05-08T09:00:00.000Z",
-            "id": "identity--985d726f-889f-48cf-b6b4-e31f562733cd",
+            "id": "identity--f8f3088c-91f7-484e-a19b-c259d05c2708",
             "modified": "2014-05-08T09:00:00.000Z",
             "name": "Fred",
             "spec_version": "2.1",
             "type": "identity"
         },
         {
             "created": "2014-05-08T09:00:00.000Z",
-            "id": "identity--6f61e6f6-c980-43be-aafb-fca7a462caf5",
+            "id": "identity--dfab1a1a-8bde-4357-b932-213bc87c20db",
             "modified": "2014-05-08T09:00:00.000Z",
             "name": "Barney",
             "spec_version": "2.1",
             "type": "identity"
         },
         {
             "contacts": [
-                "identity--985d726f-889f-48cf-b6b4-e31f562733cd",
-                "identity--6f61e6f6-c980-43be-aafb-fca7a462caf5"
+                "identity--f8f3088c-91f7-484e-a19b-c259d05c2708",
+                "identity--dfab1a1a-8bde-4357-b932-213bc87c20db"
             ],
             "created": "2014-05-08T09:00:00.000Z",
+            "discovery_methods": [
+                "Log Review",
+                "Antivirus"
+            ],
+            "external_references": [
+                {
+                    "external_id": "81645723",
+                    "source_name": "JIRA"
+                }
+            ],
             "id": "incident--1b75ee8f-44d6-819a-d729-09ab52c91fdb",
+            "intended_effects": [
+                {
+                    "description": "Bring down server",
+                    "value": "Degradation of Service"
+                }
+            ],
```

Comment:
If these are the custom test results, should these be prefixed with `x_elevator_<property_name>` (happens in many areas)

Reply:
I had this idea that "Incident" should use the "x_" prefix, but for the life of me I can't remember my reasoning - so I took it out